README.md

```diff
@@ -76,10 +77,44 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+Even if the application deals only with the English locale, it may still need
+globalization to format programming language bytes into human-understandable
+language and vice-versa in an effective and reasonable way. For example, to
+display something better than "Edited 1 minutes ago".
+
+<a name="why_globalize"></a>
+### Why Globalize?
+
+Globalize provides number formatting and parsing, date and time formatting and
+parsing, currency formatting, message formatting (ICU message format pattern),
+and plural support.
+
+Design Goals.
+
+- Leverage the Unicode CLDR data in its official JSON format.
+- Follow Unicode Technical Standard #35.
+- Allow users to load as much or as little data as they need.
+- Avoid duplicating data if using multiple i18n libraries that leverage CLDR.
+- Run in browsers and Node.js.
+- Make globalization as easy to use as jQuery.
+
+Strengths of Using Globalize.
+
+- Use the largest and most extensive i18n repository. Globalize is based on CLDR
+  data.
+- Avoid bugs caused by outdated i18n content. Globalize has no hard-coded i18n
+  data. Up-to-date CLDR data can be used directly as soon as it's released.
+- Get consistent results across different browsers. Get consistent results
+  between browsers and the server.
+- Use multiple locales simultaneously.
+
```

Comment:
@rxaviers : I really like the improvements you made to this list. Great job. Using the imperative here feels weird to me, though, since the list is referring to "strengths." One easy change would be to keep everything as is except change the verbs to declarative:

* Uses…
* Avoids…
* Gets… Gets…
* Uses…